operatorhub/openshift/release-artifacts/bundle/manifests/openshift-pipelines-operator-rh.clusterserviceversion.yaml

```diff
@@ -891,6 +891,10 @@ spec:
                   value: registry.redhat.io/openshift-pipelines/pipelines-serve-tkn-cli-rhel8@
                 - name: IMAGE_CHAINS_TEKTON_CHAINS_CONTROLLER
                   value: registry.redhat.io/openshift-pipelines/pipelines-chains-controller-rhel8@
+                - name: IMAGE_RESULTS_TEKTON_RESULTS_API
```

Comment:
you need to handle it in config file, not here

also should we add results version in csv description